### ISSUE TITLE
Image not passing `loader` and `layout` props correctly to Next Image

### ISSUE THREAD

**Issue by braaar** [simenandre/next-theme-ui#11]
**Describe the bug**
The `loader` attribute is not applied to `next-theme-ui`'s `Image` correctly. Next throws an error:

```
Unhandled Runtime Error
Error: Image with src "/customers/grid-branding.svg" is missing "loader" prop.
Read more: https://nextjs.org/docs/messages/next-image-missing-loader
```

**To Reproduce**
1. Create a Next project
2. Create a custom imageLoader and set up custom loader configuration in `next.config.js`
3. Use the `Image` component from `next-theme-ui`
4. Set the `loader` attribute on your image

**Expected behavior**
I would expect the app to load my images correctly using the custom loader, as they do just fine with the normal Next Image

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser chrome
 - Version Version 98.0.4758.80 (Official Build) (arm64)

**Additional context**
This is my loader:
```ts
export const imageLoader: ImageLoader = ({ src }) => {
  return src;
};
```
It works just fine with the standard Next Image (but then I can't use `sx`

**Comment by braaar** [comment]
It seems the same applies to the `layout` attribute.

I remvoed the loader and got a new error:
```
Error: Image with src "/customers/grid-branding.svg" must use "width" and "height" properties or "layout='fill'" property.
```

Here is my component markup:

```html
// customer is the name of a customer. The directory 'customers' contains SVG files for each customer
          <Box
            sx={{
              width: '100%',
              '& > img': { position: 'unset' },
              height: '43px',
              backgroundColor: '#ff0000',
            }}
          >
            <Image
              key={customer}
              src={`/customers/${customer}.svg`}
              layout={'fill'}
              sx={{
                objectFit: 'contain',
                width: '100%',
                position: 'relative',
                height: '100%',
                backgroundColor: '00ff00',
                color: '#0000ff',
                border: 'solid',
              }}
            />
          </Box>
```

**Comment by cobraz** [comment]
👋 

This is caused by not having a `loader` configured in your Next.js project. I've seen issues with this myself where I'm using `next export`, and not able to turn it off, even though I'm not using it.

For now, the quick fix is to stick to `next-theme-ui@1.2.0`. I had the same issue on [my own website](https://github.com/cobraz/website), which runs `next export`.

**Comment by cobraz** [comment]
Btw, if you use `next export`. This might be something for you:
https://github.com/cyrilwanner/next-optimized-images

PS: I haven't tested it!
